hack/package/deb/build.sh

```diff
@@ -63,3 +47,5 @@ apt-ftparchive release ./ > Release
 # if you want to use your gpg key, you should replace '439AE9EC' with your key
 gpg -abs --default-key 439AE9EC -o Release.gpg Release
 gpg --clearsign --default-key 439AE9EC -o InRelease Release
+
```

Comment:
Could we use a variable like DEFAULT-GPG-KEY=${GPGKEY:-"439AE9EC"} to make gpg key configurable.